lib/openstack_query/query_methods.py

```diff
@@ -127,25 +128,23 @@ def group_by(
 
     def run(
         self,
-        cloud_account: str,
+        cloud_account: Union[str, CloudDomains],
```

Comment:
If we've changed this type to be an enum either, I can't see a name.lower()?

Reply:
we do the conversion in `run_query` - because we want to limit what we do in QueryMethods